tests/Configuration/ComposerJsonParserTest.php

```diff
@@ -12,7 +12,7 @@
 final class ComposerJsonParserTest extends AbstractKernelTestCase
 {
     /**
-     * @dataProvider dataProvider @dataProvider dataProvider
+     * @dataProvider dataProvider @dataProvider dataProvider @dataProvider dataProvider @dataProvider dataProvider
```

Comment:
@TomasVotruba The `[ci-review] Rector Rectify` auto commit keep duplicating docblock